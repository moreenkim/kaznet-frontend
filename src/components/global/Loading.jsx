// This component
import React from 'react';

import './Loading.css';

function Loading() {
  return (
    <center>
      <div className="lds-ripple">
        <div />
        <div />
      </div>
    </center>
  );
}

export default Loading;
