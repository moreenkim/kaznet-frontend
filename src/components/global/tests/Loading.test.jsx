// test ExportModal
import React from 'react';
import { shallow, mount } from 'enzyme';
import toJson from 'enzyme-to-json';

import Loading from '../Loading';

describe('components/global/Loading', () => {
  it('renders without crashing', () => {
    shallow(<Loading />);
  });
  it('renders the Loading component', () => {
    const wrapper = mount(<Loading />);
    expect(toJson(wrapper)).toMatchSnapshot();
    wrapper.unmount();
  });
});
