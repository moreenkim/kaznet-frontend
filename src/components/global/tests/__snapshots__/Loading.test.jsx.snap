// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/global/Loading renders the Loading component 1`] = `
<Loading>
  <center>
    <div
      className="lds-ripple"
    >
      <div />
      <div />
    </div>
  </center>
</Loading>
`;
